arangod/Aql/Functions.cpp

```diff
@@ -4744,6 +4744,7 @@ AqlValue Functions::IsInPolygon(arangodb::aql::Query* query,
   }
 
   S2Loop loop;
+  loop.set_s2debug_override(S2Debug::DISABLE);
```

Comment:
Can we change the default debug setting at compile time, e.g. for non-maintainer builds?

Reply:
this would trigger an assertion during our tests if our own validation rules are not sufficient. For the multi-polygon the right hand rule is kind of important, because (as far as I understand it) we cannot use `initNested` to construct the S2Polygon. initNested will normalize all polygons, so wrongly oriented holes should not be a problem.

 ```  if (FLAGS_s2debug && s2debug_override_ == S2Debug::ALLOW) {
    // Note that FLAGS_s2debug is false in optimized builds (by default).
    S2_CHECK(IsValid());
  }```

Reply:
we could still set this as a flag I think